.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "oss-internship-2020/curl/curl_wrapper/curl"]
+	path = oss-internship-2020/curl/curl_wrapper/curl
+	url = https://github.com/curl/curl
+[submodule "oss-internship-2020/curl/tests/googletest"]
```

Comment:
We actually already depend on Googletest, so you should be able to use its targets in your `CMakeLists.txt` already.